service/initialise.go

```diff
@@ -67,9 +64,9 @@ func (e *ExternalServiceList) GetDatasetAPIClient(cfg *config.Config) (DatasetAP
 }
 
 // GetFileStore returns an initialised connection to file store
-func (e *ExternalServiceList) GetFileStore(cfg *config.Config, vaultClient *vault.Client) (fileStore *file.Store, err error) {
+func (e *ExternalServiceList) GetFileStore(cfg *config.Config) (fileStore *file.Store, err error) {
```

Comment:
As this is application only, is not used anywhere else, and the service is marked to be sunset in the future I will leave this as is for now rather than create a new function.